models/intermediate/intermediate.yml

```diff
@@ -13,4 +13,16 @@ models:
           combination_of_columns:
             - order_id
             - source_relation
-  - name: shopify__orders__order_refunds
\ No newline at end of file
+  - name: shopify__orders__order_refunds
```

Comment:
doesn't look related to this task, but wonder why doesn't `shopify__orders__order_refunds` have its own `dbt_utils.unique_combination_of_columns` test.

Reply:
not sure, i'd consider this out of scope for this task tho